### PR TITLE
Agrifood track 2 module release

### PR DESCRIPTION
**Pull request by ArcturusZhang**
Fixes https://github.com/Azure/sdk-release-request/issues/1656

### CODE REVIEW

**Review by chamons — APPROVED** [review]

**Comment by chamons** [comment]
Seems reasonable once build is green. Looks like you missed a link:

```
##[warning][404] broken link https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/agrifood/armagrifood
Summary of broken links:
'file:///mnt/vss/_work/1/s/sdk/agrifood/armagrifood/README.md' has 1 broken link(s):
  https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/agrifood/armagrifood
```

**Comment by ArcturusZhang** [comment]
> Seems reasonable once build is green. Looks like you missed a link:
> 
> ```
> ##[warning][404] broken link https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/agrifood/armagrifood
> Summary of broken links:
> 'file:///mnt/vss/_work/1/s/sdk/agrifood/armagrifood/README.md' has 1 broken link(s):
>   https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/agrifood/armagrifood
> ```

I must say this is inevitable - Joel can explain this much more clear, but let me try:
We are onboarding an RP now and we need to onboarding with documentations. The documentation includes a link to the package reference on `pkg.go.dev`. But the package reference cannot exist before the PR is merged and the tag is pushed.
If we choose to modify the link after the version is tagged (say v0.1.0), the customer will not see the link to the package reference when they are checking the code of v0.1.0.
Therefore for the onboarding PR of a RP, we usually merge the PR with the broken link. 
Once the PR is merged, we can click the request module button on the corresponding page on pkg.go.dev, and after it finishes, we can rerun the pipeline on main branch, and it will pass then

File: sdk/agrifood/armagrifood/zz_generated_connection.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
+// +build go1.13
```
Comment (jhendrixMSFT) [inline sdk/agrifood/armagrifood/zz_generated_connection.go:1]: Connection to be deleted.

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]
Remove generated connection.

**Comment by ArcturusZhang** [comment]
Hey @jhendrixMSFT I just removed that file and regenerated the package. Please take a look again

**Comment by ArcturusZhang** [comment]
Hi @jhendrixMSFT could you please take a look again?

**Pull request closed without merging**